### PR TITLE
RHDEVDOCS-3932 Release Notes for Build and Jenkins in 4.11

### PR DESCRIPTION
**Pull request by rolfedh**
Aligned team: Dev Tools
- For branches: 4.11 only (release notes)
- Jira: https://issues.redhat.com/browse/RHDEVDOCS-3932
- Direct link to doc preview: 
  - https://rolfedh.github.io/previews/RHDEVDOCS-3932/release_notes/ocp-4-11-release-notes.html#ocp-4-11-builds-bug-fixes
  - https://rolfedh.github.io/previews/RHDEVDOCS-3932/release_notes/ocp-4-11-release-notes.html#ocp-4-11-jenkins
  - https://rolfedh.github.io/previews/RHDEVDOCS-3932/release_notes/ocp-4-11-release-notes.html#removal-of-jenkins-images-from-install-payload
- SME review: @coreydaley @akram 
- QE review: @jitendar-singh 
- Peer review: @jc-berger 
- All reviews complete. Please merge now.

### CODE REVIEW

**Review by adambkaplan — CHANGES_REQUESTED** [review]

File: release_notes/ocp-4-11-release-notes.adoc [hunk 1]
```diff
@@ -104,6 +104,8 @@ For more information, see xref:../getting_started/openshift-overview.adoc[Gettin
 [id="ocp-4-11-jenkins"]
 === Jenkins
 
+* With this developer preview, you can run builds within a user-namespaced container by default. That is, in terms of the host, you can run a build as a non-root user. For instructions, see link:https://access.redhat.com/articles/<tbd-number>[<tbd-title>]. (link:https://issues.redhat.com/browse/BUILD-141[BUILD-141])
```
[the remark this answers is not in the document]
Comment (rolfedh) [reply release_notes/ocp-4-11-release-notes.adoc]: Thanks. Updated.

File: release_notes/ocp-4-11-release-notes.adoc [hunk 2]
```diff
@@ -941,9 +952,7 @@ See link:https://access.redhat.com/articles/6955985[Navigating Kubernetes API de
 [id="ocp-4-11-builds-bug-fixes"]
 ==== Builds
 
-[discrete]
-[id="ocp-4-11-jenkins-bug-fixes"]
-==== Jenkins
+* Before this update, Buildah printed steps to the log file by default. As a result, the contents of environment variables could be exposed by the default build settings. This update resolves the issue. The `--quiet` option is appended to Buildah builds for log levels below 2. Now Buildah build steps are no longer printed to the log when a build level below 2 is specified, or no build log level is specified. (link:https://bugzilla.redhat.com/show_bug.cgi?id=1996883[BZ#1996883])
```
Comment (coreydaley) [inline release_notes/ocp-4-11-release-notes.adoc:955]: This is not correct. That bugzilla got backed out, the new way is to set the "BUILDAH_QUIET" env var on the buildconfig

File: release_notes/ocp-4-11-release-notes.adoc [hunk 3]
```diff
@@ -17,7 +17,7 @@ Built on {op-system-base-full} and Kubernetes, {product-title} provides a more s
 {product-title} (link:https://access.redhat.com/errata/RHSA-2022:TODO[RHSA-2022:TODO]) is now available. This release uses link:https://github.com/kubernetes/kubernetes/blob/master/CHANGELOG/CHANGELOG-1.24.md[Kubernetes 1.24] with CRI-O runtime. New features, changes, and known issues that pertain to {product-title} {product-version} are included in this topic.
 //Red Hat did not publicly release {product-title} 4.10.0 as the GA version and, instead, is releasing {product-title} 4.10.3 as the GA version.
 
-{product-title} {product-version} clusters are available at https://console.redhat.com/openshift. With the {cluster-manager-first} application for {product-title} users can you to deploy OpenShift clusters to either on-premise or cloud environments.
+{product-title} {product-version} clusters are available at https://console.redhat.com/openshift. With the {cluster-manager-first} application for {product-title} users can you to deploy OpenShift clusters to either on-premises or cloud environments.
```
Comment (jc-berger) [inline release_notes/ocp-4-11-release-notes.adoc:20]: Word choice/sentence structure. 

There's some minor errors in the last sentence. Rephrase to something like:

`With the {cluster-manager-first} application for {product-title}, you can deploy OpenShift clusters to either on-premises or cloud environments.`

**Review by jc-berger — CHANGES_REQUESTED** [review]

File: release_notes/ocp-4-11-release-notes.adoc [hunk 4]
```diff
@@ -921,6 +926,12 @@ Support for deploying custom schedulers manually has been removed with this rele
...
 
+==== Removal of Jenkins images from install payload
+
+* {product-title} 4.11 moves the "OpenShift Jenkins" and "OpenShift Agent Base" images to the `ocp-tools-4` repository at `registry.redhat.io` so that Red Hat can produce and update the images outside the {product-title} lifecycle. Previously, these images were in the {product-title} install payload and the `openshift4` repository at `registry.redhat.io`. For more information, see xref:../cicd/jenkins/openshift-jenkins.adoc#openshift-jenkins[OpenShift Jenkins].
+
+* {product-title} 4.11 removes "OpenShift Jenkins Maven" and "NodeJS Agent" images from its payload. Previously, {product-title} 4.10 deprecated these images. Red Hat no longer produces these images, and they are not available from the `ocp-tools-4` repository at `registry.redhat.io`. Bug fixes and support are provided only through the end of the 4.10 and previous release lifecycles, following the link:https://access.redhat.com/support/policy/updates/openshift[{product-title} lifecycle policy]. For more information, see xref:../cicd/jenkins/openshift-jenkins.adoc#openshift-jenkins[OpenShift Jenkins].
```
Comment (gabemontero) [inline release_notes/ocp-4-11-release-notes.adoc:933]: to try and minimize the freak out @coreydaley @akram @rgormley I think we should also clarify here that a) the jenkins maven and nodejs imagestreams installed from 4.10 and earlier will not be removed if you upgrade to 4.11, b) openshift engineering will support those 4.10 etc. images on 4.11+ cluster through the end of the 4.10 maintenance window

what do you all think?

File: release_notes/ocp-4-11-release-notes.adoc (hunk 2 above)
Comment (rolfedh) [reply release_notes/ocp-4-11-release-notes.adoc]: Replaced this RN with the one for https://bugzilla.redhat.com/show_bug.cgi?id=2099991. Thanks.

File: release_notes/ocp-4-11-release-notes.adoc (hunk 3 above)
Comment (rolfedh) [reply release_notes/ocp-4-11-release-notes.adoc:20]: Excellent! Thanks!

File: release_notes/ocp-4-11-release-notes.adoc (hunk 4 above)
Comment (rolfedh) [reply release_notes/ocp-4-11-release-notes.adoc:933]: I'll work that into this release note. Then, please let me know what you think. Thanks.

Comment (rolfedh) [reply release_notes/ocp-4-11-release-notes.adoc]: @gabemontero @coreydaley @akram @rgormley PTAL at the updated version.

File: release_notes/ocp-4-11-release-notes.adoc [hunk 5]
```diff
@@ -921,6 +926,16 @@ Support for deploying custom schedulers manually has been removed with this rele
...
+
+* {product-title} 4.11 removes "OpenShift Jenkins Maven" and "NodeJS Agent" images from its payload. Previously, {product-title} 4.10 deprecated these images. Red Hat no longer produces these images, and they are not available from the `ocp-tools-4` repository at `registry.redhat.io`.
++
+However, upgrading to {product-title} 4.11 does not remove "OpenShift Jenkins Maven" and "NodeJS Agent" images from 4.10 and earlier releases. And Red Hat provides bug fixes and support for these images on through the end of the 4.10 release lifecycle, in accordance with the link:https://access.redhat.com/support/policy/updates/openshift[{product-title} lifecycle policy].
++
+For more information, see xref:../cicd/jenkins/openshift-jenkins.adoc#openshift-jenkins[OpenShift Jenkins].
```
Comment (rolfedh) [inline release_notes/ocp-4-11-release-notes.adoc:937]: > to try and minimize the freak out @coreydaley @akram @rgormley I think we should also clarify here that a) the jenkins maven and nodejs imagestreams installed from 4.10 and earlier will not be removed if you upgrade to 4.11, b) openshift engineering will support those 4.10 etc. images on 4.11+ cluster through the end of the 4.10 maintenance window

@gabemontero @coreydaley @akram @rgormley PTAL at this updated version.

Comment (coreydaley) [reply release_notes/ocp-4-11-release-notes.adoc:1066]: That looks good to me. Thanks!

**Review by jc-berger — APPROVED** [review]
/lgtm

File: release_notes/ocp-4-11-release-notes.adoc (hunk 1 above)
[the remark this answers is not in the document]
Comment (rolfedh) [reply release_notes/ocp-4-11-release-notes.adoc]: This feature has been postponed to 4.12. Removed this release note.

**Comment by coreydaley** [comment]
/assign @jitendar-singh 
Please review and approve.

**Comment by Preeticp** [comment]
Changes requested by Adam are not relevant to the release, hence going ahead and merging this PR despite the change request.

**Pull request merged**